scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
Quick check: let's say I set up a different git project on the same machine. Does this hook run?

(If so, that's a problem.)